internal/pkg/errors/error_message.go

```diff
@@ -495,6 +497,7 @@ const (
 	// General
 	DeleteResourceErrorMsg        = `failed to delete %s "%s": %v`
 	DeleteResourceConfirmErrorMsg = `input does not match "%s"`
+	ListResourceSuggestions       = "List available %ss with `confluent %s list`."
```

Comment:
We should probably call the `plural()` function, since there might be edge cases.
```suggestion
	ListResourceSuggestions       = "List available %s with `confluent %s list`."
```